pkgs/os-specific/linux/rtl8812au/default.nix

```
@@ -46,10 +44,10 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Driver for Realtek 802.11ac, rtl8812au, provides the 8812au mod";
-    homepage = "https://github.com/gordboy/rtl8812au-5.9.3.2";
+    homepage = "https://github.com/morrownr/8812au-20210629";
     license = licenses.gpl2Only;
     platforms = platforms.linux;
     maintainers = with maintainers; [ fortuneteller2k ];
-    broken = kernel.kernelOlder "4.10" || kernel.kernelAtLeast "5.15" || kernel.isHardened;
+    broken = kernel.kernelOlder "4.10" || kernel.isHardened;
```

```suggestion
    broken = kernel.kernelOlder "4.10" || kernel.isHardened;
```
The broken marking here is imprecise. Needs to run nixpkgs-review (without broken), re-catalog the broken packages, and update this.